website/docs/r/launch_configuration.html.markdown

```diff
@@ -10,6 +10,8 @@ description: |-
 
 Provides a resource to create a new launch configuration, used for autoscaling groups.
 
+!> **WARNING:** The use of launch configurations is discouraged in favour of launch templates. Read more in the [documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/launch-configurations.html).
```

Comment:
```suggestion
!> **WARNING:** The use of launch configurations is discouraged in favour of launch templates. Read more in the [AWS EC2 Documentation](https://docs.aws.amazon.com/autoscaling/ec2/userguide/launch-configurations.html).
```

This looks good, thank you @jramosf! 🚀  I think we should make a small change to make it clear to readers that its AWS discouraging its use rather than Terraform.